stable/appmesh-inject/values.yaml

```diff
@@ -43,6 +43,8 @@ tolerations: []
 
 affinity: {}
 
+podAnnotations: {}
```

Comment:
This is actually the only change you needed to make :) The error was complaining about there being no "podAnnotations" key in the Values hashmap.

Reply:
So, I had changed `podAnnotations` only and tried to install, but the same error occurred :(